src/core/panel/plugin-panel.tsx

```diff
@@ -60,7 +64,7 @@ export abstract class UIPPluginPanel extends UIPPlugin {
   /** Creates resize area element */
   @memoize()
   protected get $resize(): HTMLElement {
-    return (<div class="uip-plugin-resize-bar" />) as HTMLElement;
+    return (<div className='uip-plugin-resize-bar' />) as HTMLElement;
```

Comment:
Not a request, just an overall note: JSX Dom works according to progressive Preact but not clear React API, so both `class` and `className` are totally allowed.